pkg/sanity/sanity.go

```diff
@@ -136,3 +137,23 @@ func loadSecrets(path string) (*CSISecrets, error) {
 
 	return &creds, nil
 }
+
+var uniqueSuffix = "-" + pseudoUUID()
+
+// pseudoUUID returns a unique string generated from random
+// bytes, empty string in case of error.
+func pseudoUUID() string {
+	b := make([]byte, 8)
+	if _, err := rand.Read(b); err != nil {
```

Comment:
Do you need to seed the random generator at the beginning?

Reply:
This is `crypto/rand`, which doesn't need to be seeded.

One could argue that the normal `rand` is good enough, but then it would need to be seeded. I don't care either way.